src/main/java/nl/knaw/dans/easy/sword2examples/ContinuedDeposit.java

```diff
@@ -97,6 +98,7 @@ public static URI depositPackage(File bag, IRI colIri, String uid, String pw, in
 
         // 4. Get the statement URL. This is the URL from which to retrieve the current status of the deposit.
         System.out.println("Retrieving Statement IRI (Stat-IRI) from deposit receipt ...");
+        System.out.println(bodyText); // TODO get the target of the redirected content
```

Comment:
It is a http/https issue

The first response body returns
```
<entry xmlns="http://www.w3.org/2005/Atom">
    <generator uri="http://www.swordapp.org/" version="2.0"/>
    <id>http://deasy.dans.knaw.nl/sword2/container/6038e964-c990-4e64-8da6-9cad9af9c8d5</id>
    <link href="http://deasy.dans.knaw.nl/sword2/container/6038e964-c990-4e64-8da6-9cad9af9c8d5" rel="edit"/>
    <link href="http://deasy.dans.knaw.nl/sword2/container/6038e964-c990-4e64-8da6-9cad9af9c8d5" rel="http://purl.org/net/sword/terms/add"/>
    <link href="http://deasy.dans.knaw.nl/sword2/media/6038e964-c990-4e64-8da6-9cad9af9c8d5" rel="edit-media"/>
    <packaging xmlns="http://purl.org/net/sword/terms/">http://purl.org/net/sword/package/BagIt</packaging>
    <link href="http://deasy.dans.knaw.nl/sword2/statement/6038e964-c990-4e64-8da6-9cad9af9c8d5" rel="http://purl.org/net/sword/terms/statement" type="application/atom+xml; type=feed"/>
    <treatment xmlns="http://purl.org/net/sword/terms/">[1] unpacking [2] verifying integrity [3] storing persistently</treatment>
    <verboseDescription xmlns="http://purl.org/net/sword/terms/">received successfully: bag.zip.1; MD5: 698abb2a4bc79b713916b98ae00745d1</verboseDescription>
</entry>
```

The redirect response body:
```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html>
    <head>
        <title>301 Moved Permanently</title>
    </head>
    <body>
        <h1>Moved Permanently</h1>
        <p>The document has moved <a href="https://deasy.dans.knaw.nl/sword2/container/6038e964-c990-4e64-8da6-9cad9af9c8d5">here</a>.
        </p>
    </body>
</html>
```

Reply:
Looks like we need to change https://github.com/DANS-KNAW/easy-dtap/blob/cf8d85b18f6b150601725b4cfea5b3323db3dc76/provisioning/host_vars/deasy.yml#L20